var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -255,6 +255,14 @@ def configure_args(self):
             else:
                 config_args.append('--disable-mpi-thread-multiple')
 
+        # CUDA support
+        if spec.satisfies('@1.6:'):
+	    if '+cuda' in spec:
+                config_args.append('--with-cuda={0}'.format(spec['cuda'].prefix))
+                config_args.append('--with-cuda-libdir={0}/lib64'.format(spec['cuda'].prefix))
```

Comment:
I'm not entirely sure if its always in lib64, cuda only has 64bit downloads for linux so I would assume that its safe to have it like this. I can make the change spec['cuda'].libs.directories